test/apiv2/python/rest_api/test_v2_0_0_container.py

```diff
@@ -359,6 +361,55 @@ def test_memory(self):
         self.assertEqual(2000, out["HostConfig"]["MemorySwap"])
         self.assertEqual(1000, out["HostConfig"]["Memory"])
 
+    def test_host_config_port_bindings(self):
+        # create an image with ports 8081 and 8080 exposed
+        tar_dockerfile = io.BytesIO()
+        with tarfile.open(fileobj=tar_dockerfile, mode="w:gz") as tar:
+            my_content = "from alpine:latest\nexpose 8081\nexpose 8080".encode('utf-8')
+            tf = tarfile.TarInfo("Dockerfile")
+            tf.size = len(my_content)
+            tar.addfile(tf, io.BytesIO(my_content))
+        tar_dockerfile.seek(0)
+        r = requests.post(
+            self.podman_url
+            + "/v1.40/build?t=alpine_exposed",
+            timeout=15,
+            headers={'Content-type': 'application/tar'},
+            data=tar_dockerfile
+        )
+        self.assertEqual(r.status_code, 200, r.text)
```

Comment:
test speed is important, there is no need to build an extra image here, you should be able to expose ports on the create call: https://docs.docker.com/engine/api/v1.41/#tag/Container/operation/ContainerCreate
see the ExposedPorts field